cardInfoAPI-Level1.yaml

```
@@ -0,0 +1,1402 @@
+      title: Mcc description
+      type: string
+      maxLength: 100
+      examples:
+        - Telco
+      description: Description for the mcc.
+
+    TerminalReference:
+      title: Terminal reference (also known as terminal id)
+      type: string
+      maxLength: 8
+      examples:
+        - '12345678'
+      description: Identifier for the terminal which has read the card data.
+
+    CardAcceptorReference:
+      title: Card acceptor reference
+      type: string
+      maxLength: 15
+      examples:
+        - '87040'
+      description: Identifier for the merchant who requested the transaction.
+
+    AcquirerReference:
+      title: Acquirer reference (also known as acquirer id)
+      type: string
+      maxLength: 11
+      examples:
+        - '12345678901'
+      description: Identifies the acquirer who submitted the transaction; not provided by all Card Service Providers.
+
+    Channel:
+      type: string
+      enum:
+        - standard
+        - ecommerce
+      examples:
+        - standard
+      description: Channel that was used by the cardholder for executing the transaction, allows the identification of eCommerce transactions.
+
+    TransactionCategory:
+      title: Transaction category
+      type: string
+      maxLength: 50
+      examples:
+        - '100: Cash withdrawal'
+        - '103: ATM operator refund'
+        - '117: P2P payment charge sender'
+        - '118: P2P payment credit recipient'
+        - merchant
+        - fee
+        - atm
+      description: Category of the transaction, defined by the Card Service Provider.
+
+    CardholderReference:
+      title: Cardholder reference
+      type: string
+      maxLength: 50
+      examples:
+        - Peter Schweizer
+      description: Reference to the cardholder who owns the card which was used for the transaction; not provided by all Card Service Providers.
+
+    RelatedTransactionId:
+      title: Transaction reference
+      type: string
+      format: uuid
+      maxLength: 50
+      examples:
+        - 9e19df54-47cb-494b-84dc-1c126e95a05c
+      description: Reference to another transaction or authorization that is linked to the current one, e.g. for separately charged fees, chargeback credits.
+
+    FeeName:
+      title: Fee name
+      type: string
+      maxLength: 50
+      examples:
+        - transactionTipAmount
+        - transactionSurchargeAmount
+      description: Name of the fee that is charged for the transaction; value is defined by the Issuer.
+
+    FeePercentage:
+      title: Fee percentage
+      type: number
+      examples:
+        - 3.5
+      description: >
+        Percentage value of the transactionAmount for the fee type; not provided by all Card Service Providers.
+        Values are provided to be compatible with the % notation. For example a value 3.5 is to be interpreted as 3.5% (= 0.035 as decimal).
+
+    FeeAmount:
+      title: Fee amount
+      type: number
+      examples:
+        - 5.00
```

this example is wrong as type number will not render the fraction, and this is problematic for monetary values.

```suggestion
        - 5
```

---

Will be adjusted according to your feedback above.